### PR TITLE
Feature/ws fetching with fallback

### PR DESCRIPTION
**Pull request by LukasDeco**

### CODE REVIEW

File: components/Orders/ProposalOpenOrderRow.tsx [hunk 1]
```diff
@@ -107,6 +111,22 @@ export function ProposalOpenOrderRow({ order }: { order: OpenOrdersAccountWithKe
     try {
       setIsEditing(true);
       await sender.send(txs);
+      // TODO, once edit working as expected, enable this
```
Comment (LukasDeco) [inline components/Orders/ProposalOpenOrderRow.tsx:114]: @R-K-H I have had a weird experience with this where it reliably does not edit the order, but rather just closes the order and then allows me to settle in the unsettled tab. I assume this isn't desired functionality? Have you seen this?

Comment (R-K-H) [reply components/Orders/ProposalOpenOrderRow.tsx:114]: Yea I don't think edit ever worked. It was tossed in without full testing, I think it was expected SOMEONE would pick it up, and when we were a collective of like 20 people just messing around on weekends, it worked for us ;)

Comment (LukasDeco) [reply components/Orders/ProposalOpenOrderRow.tsx:114]: haha did it really work during those weekends? I'm pretty sus at this point. But okay I'll take your word for it.

**Review by R-K-H — APPROVED** [review]
LGTM!

**Pull request merged**